### PR TITLE
Eg ensemble measures add params

### PR DESCRIPTION
**Pull request by egolol**
The proposed solution to parameter bug in ensemble and comparison. In GUI the user has to click the item MeasureParam to set the core params used in the ensemble.

### CODE REVIEW

**Review by annacanal — COMMENTED** [review]
* When having several parameters in a measure and some are not specified in the set, they are set as 'NoValue', and then the measure does not see that variables empty. Change 'NoValue' for empty
* As we talked, the rules/parameters do not appear from the GUI without doing .set() from command line

**Comment by egolol** [comment]
> * When having several parameters in a measure and some are not specified in the set, they are set as 'NoValue', and then the measure does not see that variables empty. Change 'NoValue' for empty
> * As we talked, the rules/parameters do not appear from the GUI without doing .set() from command line

Solved

**Review by annacanal — CHANGES_REQUESTED** [review]
It gives errors when calculating the measure, in the prop checks, when prop = []

**Review by annacanal — APPROVED** [review]

File: braph2genesis/measures/_CommunityStructure.gen.m [hunk 1]
```diff
@@ -146,7 +146,7 @@
             Ci_corrected(n_perm) = Ci;  % return order of nodes to the order used at the input stage.
             Ci = Ci_corrected;  % output corrected community assignments
             
-            mo.quality_function = Q;  % save normalized quality function/modularity
+            m.set('quality_function', Q);   % save normalized quality function/modularity
```
Comment (giovannivolpe) [inline braph2genesis/measures/_CommunityStructure.gen.m:149]: Capitalize property names

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

File: braph2genesis/measures/_CommunityStructure.gen.m (hunk 1 above)
Comment (annacanal) [reply braph2genesis/measures/_CommunityStructure.gen.m]: Done

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**